app/views/movies/show.html.slim

```diff
@@ -1,10 +1,38 @@
 - provide(:title, @movie[:title])
 
-h1
-  = @movie[:title]
-
-.index-grid
-  img.thumbnail.homepage src="#{@movie[:image_url]}"
-  .row
-    h5
-      = @movie[:description]
+.row.flex-container.card.small-11.small-centered.medium-10.medium-centered.large-7.large-centered.xlarge-6.xxlarge-5
```

Comment:
Try to seperate the markup so we know what it looks like on the front end
.row.flex-container.card
[tab].small-11.small-centered.medium-10.medium-centered.large-7.large-centered.xlarge-6.xxlarge-5